vss-extension.json

```diff
@@ -1,9 +1,9 @@
 {
   "manifestVersion": 1,
-  "id": "synopsys-detect-qa",
-  "version": "9.0.0",
-  "name": "Synopsys Detect QA",
-  "publisher": "synopsys-detect",
+  "id": "blackduck-detect-qa",
+  "version": "10.0.0",
+  "name": "Black Duck Detect QA",
+  "publisher": "blackduck-detect",
```

Comment:
@shantyk The publisher based on entry already created in marketplace should be just "blackduck"